### PR TITLE
Updating things that really have should been in main and the import and fixes tbh

### PR DESCRIPTION
**Pull request by AstralTechSolutions**
Basically I was just stupid and realized that the `whenPressed` method was deprecated and should be replaces with `onTrue` or `whenTrue` Also the gyro is no longer inverted so that got changed in constants.

### CODE REVIEW

**Comment by AstralTechSolutions** [comment]
Oh yeah @SilverBulletO1 the invert gyro is the change I walked you through on the phone because swerve would not turn right. I just missed it with the initial imports to 2023.

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -55,7 +55,7 @@ public RobotContainer() {
    */
   private void configureButtonBindings() {
     /* Driver Buttons */
-    zeroGyro.whenPressed(new InstantCommand(() -> swerveSubsystem.zeroGyro()));
+    zeroGyro.onTrue(new InstantCommand(() -> swerveSubsystem.zeroGyro()));
```
Comment (daonguyenk06) [inline src/main/java/frc/robot/RobotContainer.java:58]: Can you leave a comment on what onTrue does instead of whenPressed? That might help some of us.

**Review by 25kingk — DISMISSED** [review]
You likely need more comments. While I understand the change, others might not and explaining some of the more complex or confusing code would probably be a good thing.

File: src/main/java/frc/robot/Constants.java [hunk 2]
```diff
@@ -14,7 +14,7 @@ public final class Constants {
 
     public static final class Swerve {
         public static final int pigeonID = 0;  //Can ID for Pigeon IMU. 
-        public static final boolean invertGyro = true; // Is the GYRO Upside Down?
+        public static final boolean invertGyro = false; // Is the GYRO Upside Down?
```
Comment (42daha) [inline src/main/java/frc/robot/Constants.java:17]: Does the pidgeon care?  Having a comment to explain would be nice.

**Review by 42daha — APPROVED** [review]

**Review by Trex1620 — APPROVED** [review]

**Review by Trex1620 — APPROVED** [review]

**Review by LemonHead2114 — APPROVED** [review]
I

**Pull request merged**